README.md

```diff
@@ -16,9 +16,9 @@ Read more about [SUIT's design principles](https://github.com/suitcss/suit/).
 * `Button` - [core] The core button component
 * `is-disabled` - [state] For disabled-state button styles (themes)
 
-N.B. You must also include the `disabled` attribute on `button` and `input`
-elements. For `a` elements, you should remove the `href` attribute and prevent
-JavaScript event handlers from firing.
+N.B. You must also include the `disabled` attribute on `button` elements. For
+`a` elements, you should remove the `href` attribute and prevent JavaScript
```

Comment:
Why not use a `span` then? I try to advocate this as much as possible but it is a tough one to win :)